src/messages/responses/cache-list-push-front.ts

```diff
@@ -0,0 +1,26 @@
+import {SdkError} from '../../errors/errors';
+import {ResponseBase} from './response-base';
+import {applyMixins, ErrorBody} from '../../errors/error-utils';
+
+export abstract class Response extends ResponseBase {}
+
+export class Success extends Response {
+  private readonly _list_length: number;
+  constructor(list_length: number) {
+    super();
+    this._list_length = list_length;
```

Comment:
Do we want to make sure if `list_length` is within the range of `number` type like [this](https://github.com/momentohq/client-sdk-dotnet-incubating/blob/main/src/Momento.Sdk.Incubating/Responses/CacheListPushFrontResponse.cs#L17)?